README.md

```diff
@@ -8,7 +8,7 @@
 [license-badge]: https://img.shields.io/github/license/juspay/hyperswitch
 [license-url]: https://github.com/juspay/hyperswitch/blob/main/LICENSE
 
-hyperswitch is a **_Payment Switch_** that lets you connect with **multiple payment processors with a single API integration**.
+hyperswitch is an open-source **_Payment Switch_** to make your Payments fast, reliable and affrodable. It lets you connect with **multiple payment processors with a single API integration**.
```

Comment:
`affrodable` -> `affordable`

Reply:
is there an external reference for "Payment Switch" that this can link to?